content/en/docs/reference/access-authn-authz/service-accounts-admin.md

```diff
@@ -10,79 +10,84 @@ weight: 50
 ---
 
 {{% capture overview %}}
-This is a Cluster Administrator guide to service accounts. It assumes knowledge of
+This is a Cluster Operator guide to service accounts. It assumes knowledge of
 the [User Guide to Service Accounts](/docs/user-guide/service-accounts).
 
-Support for authorization and user accounts is planned but incomplete.  Sometimes
-incomplete features are referred to in order to better describe service accounts.
 {{% /capture %}}
 
 {{% capture body %}}
-## User accounts vs service accounts
-
-Kubernetes distinguishes between the concept of a user account and a service account
-for a number of reasons:
-
-  - User accounts are for humans. Service accounts are for processes, which
-    run in pods.
-  - User accounts are intended to be global. Names must be unique across all
-    namespaces of a cluster, future user resource will not be namespaced.
-    Service accounts are namespaced.
-  - Typically, a cluster's User accounts might be synced from a corporate
-    database, where new user account creation requires special privileges and
-    is tied to complex business processes. Service account creation is intended
-    to be more lightweight, allowing cluster users to create service accounts for
-    specific tasks (i.e. principle of least privilege).
-  - Auditing considerations for humans and service accounts may differ.
-  - A config bundle for a complex system may include definition of various service
-    accounts for components of that system.  Because service accounts can be created
-    ad-hoc and have namespaced names, such config is portable.
+## Users vs service accounts
+
+Kubernetes [distinguishes](/docs/reference/access-authn-authz/authentication/#users-in-kubernetes)
+between the concept of a user and a service account for a number of reasons:
+
+  - Users represent humans. Service accounts are for processes, which
+    run in {{< glossary_tooltip text="pods" term_id="pod" >}}.
```

Comment:
- I'm not sure this change helps clarify a user account vs. service account.
- are italics needed?